scripts/password_recovery/PasswordRecovery.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+
+	"golang.org/x/crypto/bcrypt"
+)
+
+var (
+	password = flag.String("password", "password", "Prefix for all Docker images and volumes")
+)
+
+func main() {
+	flag.Parse()
+
+	args := os.Args[1:]
+	if len(args) > 0 && *password == "password" {
+		// didn't use flag
+		inferred := args[len(args)-1]
+		password = &inferred
+	}
+
+	fmt.Printf("generating hash for password %s: \n", string(*password))
```

Comment:
I don't think it matters. it's a good check that we didn't make a mistake when inputting it